covid-19-support/src/components/ResourceMap.vue

```diff
@@ -78,6 +93,9 @@ export default {
       center: latLng(this.centroid.lat, this.centroid.lng),
       zoom: 10,
       showParagraph: true,
+      showError: false,
+      errorMessage: '',
+      locationHelpUrl: 'https://support.google.com/maps/answer/2839911?hl=en&authuser=0&visit_id=1589939482955-302781235128857673&rd=1',
```

Comment:
I don't think we should be linking off to this page as it would be very confusing to end users, we need to have our own help information show up if the user clicks to "Learn More" I suggest you show a Dialog ( in the same way the About Us information currently shows and then we need to write some help text. Feel free to write something short for now, just worry about getting the functionality working )

Reply:
I changed this to pop up a modal now. Let me know if this looks good